fabric-network/test/impl/event/blocklistener.spec.ts

```diff
@@ -110,6 +113,31 @@ describe('block listener', () => {
 		});
 	}
 
+	function addTransaction(event:EventInfo): EventInfo {
+		event.block.data = new fabproto6.common.BlockData();
+		event.block.data.data.push(newEnvelope(new fabproto6.protos.Transaction(), 'txn1'));
+		event.block.metadata = new fabproto6.common.BlockMetadata();
+		event.block.metadata.metadata = [];
+		event.block.metadata.metadata[fabproto6.common.BlockMetadataIndex.TRANSACTIONS_FILTER] = new Uint8Array(10);
+		return event;
+	}
+
+	function newEnvelope(transaction: any, transactionId?: string): any {
+		const channelHeader :any = {};
+		channelHeader.type = fabproto6.common.HeaderType.ENDORSER_TRANSACTION;
+		channelHeader.tx_id = transactionId;
+		const timestamp = new fabproto6.google.protobuf.Timestamp({seconds:1630075029083});
+		channelHeader.timestamp = new Date(timestamp.seconds as number).toISOString();
```

Comment:
The block decoder looks to decode the protobuf timestamp to a date string like you've done here, so that's good. So we know the value to expect in the timestamp field, I think we'd be better to declare a constant in the `describe` block, something like `const timestamp = new Date()`, and then populate the timestamp with `timestamp.toISOString()`